suitcase/tiff_series/__init__.py

```diff
@@ -16,13 +16,13 @@
 del get_versions
 
 
-def export(gen, directory, file_prefix='{uid}-', stack_images=True,
-           bigtiff=False, byteorder=None, imagej=False, **kwargs):
+def export(gen, directory, file_prefix='{uid}-', bigtiff=False, byteorder=None,
```

Comment:
should this default change to `'{start.uid'}'?

Reply:
done